src/main/java/core/basesyntax/service/operations/BalanceOperationsHandler.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.Operations;
+
+public class BalanceOperationsHandler implements OperationsHandler {
+
+    @Override
+    public Operations getOperation() {
+        return Operations.BALANCE;
+    }
+
+    @Override
+    public int getOperationsResult(int result, int quantity) {
+        return result + quantity;
```

Comment:
The `getOperationsResult` method should not directly add the quantity to the result. For a balance operation, you might want to set the result to the quantity instead, as it represents setting the new balance. If adding is the intended behavior, please disregard this comment.